v8/src/base/platform/platform-win32.cc

```diff
@@ -656,7 +656,7 @@ int OS::SNPrintF(char* str, int length, const char* format, ...) {
 
 
 int OS::VSNPrintF(char* str, int length, const char* format, va_list args) {
-#if defined(COBALT)
+#if defined(V8_OS_STARBOARD)
```

Comment:
Are `STARBOARD` and `V8_OS_STARBOARD` interchangeable under `v8`?
Do we have a scheme for which to use when?

Reply:
Same comment as above - it should be consistent with upstream package conventions in upstream code, where practical. We use `V8_OS_STARBOARD` in most of V8 code, so this is just cleaning up a stray leftover.